lib/hedge_fund_interview/prisms/utils.ex

```diff
@@ -1,6 +1,8 @@
 defmodule HedgeFundInterview.Prisms.Utils do
+  @agent_api_endpoint "https://agents-api.spectrallabs.xyz/"
```

Comment:
hardcoding it in the code doesn't seem great. This is an Elixir app, not a library anymore. Why are we not using normal application config with `runtime.exs` and proper env var loading with Dotenvy?